examples/plot_anomaly_comparison.py

```diff
@@ -2,30 +2,25 @@
 ============================================================================
 Comparing anomaly detection algorithms for outlier detection on toy datasets
 ============================================================================
-
 This example shows characteristics of different anomaly detection algorithms
 on 2D datasets. Datasets contain one or two modes (regions of high density)
 to illustrate the ability of algorithms to cope with multimodal data.
-
```

Comment:
I don't think the changes to this file are warranted. Better to stick to one goal in the pull request. Please revert changes to this file.

Reply:
Sure, got it. Thank you.